packages/common/src/extensions/slickColumnPicker.ts

```diff
@@ -221,8 +221,15 @@ export class SlickColumnPicker {
   protected repositionMenu(event: DOMMouseOrTouchEvent<HTMLDivElement> | SlickEventData): void {
     const targetEvent: MouseEvent | Touch = (event as TouchEvent)?.touches?.[0] ?? event;
     if (this._menuElm) {
+      const availableWidth = window.innerWidth - (this._menuElm.getBoundingClientRect().right + targetEvent.pageX)
+      let leftPosition = 10
+
+      if (availableWidth < 0) {
+        leftPosition = leftPosition + 60 + Math.abs(availableWidth)
```

Comment:
where does the 60 come from? it's not very clear what this calculation does. It's also not covered by unit test, it would be need to be covered with a new test. I also just don't understand what this does, please provide either a print screen or animated gif of what this fixes